src/components/PlantCardPreview/PlantCardPreview.tsx

```diff
@@ -46,7 +41,7 @@ export const PlantCardPreview = ({ plantInfo }: PlantCardPreviewProps) => {
 			<FastImage
 				style={styles.plantImage}
 				source={{
-					uri: photo?.urlPath || getPlantImagePlaceholder(),
+					uri: photo?.urlPath || Image.resolveAssetSource(placeholderImage).uri,
```

Comment:
I think we can get this uri only once, outside of component function scope, so as not to load this image on each re-render